dns/unbound-plus/src/opnsense/mvc/app/controllers/OPNsense/Unboundplus/forms/dnsbl.xml

```diff
@@ -11,6 +11,14 @@
         <type>select_multiple</type>
         <help>Select which kind of DNSBL you want to use.</help>
     </field>
+    <field>
+        <id>dnsbl.lists</id>
+        <label>URLs of blacklists</label>
```

Comment:
the file seems to start usually words uppercase, so I would change that like this to look like the others:
```suggestion
        <label>URLs of Blacklists</label>
```